### PR TITLE
docs(example): framer-motion

### PR DESCRIPTION
**Pull request by Anishpras**

### CODE REVIEW

File: examples/with-framer-motion/app/routes/index.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { motion } from "framer-motion";
+export default function Index() {
+  return (
+    <div style={{ fontFamily: "system-ui, sans-serif", lineHeight: "1.4" }}>
+      <motion.h1 animate={{ scale: 0.5 }} transition={{ duration: 2 }}>
+        Welcome to Remix + Framer Motion
+      </motion.h1>
+      <ul>
```
Comment (kentcdodds) [inline examples/with-framer-motion/app/routes/index.tsx:8]: Let's remove all this stuff that doesn't have anything to do with the example.

File: examples/with-framer-motion/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Welcome to Remix + framer-motion!
```
Comment (kentcdodds) [inline examples/with-framer-motion/README.md:1]: Could you update this README to conform to the template for examples? (You can copy the template directory when creating examples to help ensure the example is simple and consistent).

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Thanks! Just a few suggestions. Also, please move this from "with-framer-motion" to simply "framer-motion". The "with-" prefix is unnecessary.

**Comment by Anishpras** [comment]
@kentcdodds I have changed as requested. Could you please review it again? Thanks.

**Comment by Anishpras** [comment]
> @kentcdodds I have changed as requested. Could you please review it again? Thanks.

I have changed a few more things. Thank you for your insightful reviews.

**Review by kentcdodds — APPROVED** [review]
Thanks!

**Pull request merged**